src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -25,17 +21,8 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsLinux)' == 'true'">
```

Comment:
I can't remember from the top of my head where I was defining TargetsLinux and TargetsWindows (I believe it was on root's Directory.Build.Props) but we should remove that as well.

Reply:
Yes, removed this one.